pkg/controllers/statusaggregator/plugins/deployment.go

```diff
@@ -39,9 +39,13 @@ func NewDeploymentPlugin() *DeploymentPlugin {
 }
 
 func (receiver *DeploymentPlugin) AggregateStatues(
+	ctx context.Context,
 	sourceObject, fedObject *unstructured.Unstructured,
 	clusterObjs map[string]interface{},
 ) (*unstructured.Unstructured, bool, error) {
+	logger := klog.FromContext(ctx).WithValues("source-object-name", sourceObject.GetName(), "status-aggregator-plugin", "deployments")
```

Comment:
`source-object-name` should have been added under `object`.

Reply:
get, object is added during logger initialization